lib/active_storage_support/base64_attach.rb

```diff
@@ -3,14 +3,33 @@ module ActiveStorageSupport
   module Base64Attach
     module_function
 
-    def attachment_from_base64(data:, filename: Time.current.to_i.to_s)
-      return data unless data.is_a?(String) && data.strip.start_with?('data')
+    def attachment_from_data(data:, filename: Time.current.to_i.to_s, content_type: nil)
+      case data
+      when Hash
```

Comment:
why all these changes? wasn't it working just returning `data` if it was not a string?

Reply:
But you are not supporting `user.avatar.attach(params[:avatar])`. You are supporting `user.avatar.attach(data: params[:avatar])`. But why would you allow that? I don't think it adds any kind of value, keywords arguments are already a `Hash` so when you call `user.avatar.attach(data: params[:avatar], filename: 'filename')` this is exactly the same as `user.avatar.attach({ data: params[:avatar], filename: 'filename' })`.